### ISSUE TITLE
"Stop build on Jenkins" failed with exception

### ISSUE THREAD

**Issue by jacekgajek** [MCMicS/jenkins-control-plugin#455]
```
java.lang.IllegalArgumentException: URL is malformed
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.handleException(UrlBuilder.java:158)
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.buildUrl(UrlBuilder.java:219)
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.createStopBuildUrl(UrlBuilder.java:111)
	at org.codinjutsu.tools.jenkins.logic.RequestManager.stopBuild(RequestManager.java:247)
	at org.codinjutsu.tools.jenkins.logic.RequestManager.stopBuild(RequestManager.java:341)
	at org.codinjutsu.tools.jenkins.view.action.StopBuildAction$1.run(StopBuildAction.java:53)
	at org.codinjutsu.tools.jenkins.logic.JenkinsBackgroundTask.run(JenkinsBackgroundTask.java:41)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:429)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsynchronously$6(CoreProgressManager.java:480)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:252)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:173)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:252)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.net.MalformedURLException: no protocol: stopstop
	at java.base/java.net.URL.<init>(URL.java:674)
	at java.base/java.net.URL.<init>(URL.java:569)
	at java.base/java.net.URL.<init>(URL.java:516)
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.buildUrlNotNull(UrlBuilder.java:227)
	at org.codinjutsu.tools.jenkins.logic.UrlBuilder.buildUrl(UrlBuilder.java:217)
	... 25 more

```

**Comment by MCMicS** [comment]
Which plugin version do you use?
please mention at least the version if file an issue

**Comment by MCMicS** [comment]
it seem you use 0.13.16

can you please try EAP for 0.13.17: https://plugins.jetbrains.com/plugin/6110-jenkins-control/versions/eap/292490

**Comment by MCMicS** [comment]
released with [0.13.17](../releases/tag/v0.13.17)
